thirdparty/build-thirdparty.sh

```diff
@@ -501,7 +501,7 @@ build_lzo2() {
 build_bzip() {
     check_if_source_exist $BZIP_SOURCE
     cd $TP_SOURCE_DIR/$BZIP_SOURCE
-
+    CFLAGS="${CFLAGS} -fPIC"
```

Comment:
`-fPIC` should be included in the global cflags, which shall take effect by default if not explicitly overwritten.